src/cljs/commiteth/bounties.cljs

```diff
@@ -23,32 +52,48 @@
          balance-eth  :balance-eth
          value-usd    :value-usd
          claim-count  :claim-count} bounty
-        full-repo  (str owner "/" repo-name)
-        repo-url   (str "https://github.com/" full-repo)
-        repo-link  [:a {:href repo-url} full-repo]
-        issue-link [:a
-                    {:href (issue-url owner repo-name issue-number)}
-                    issue-title]]
-    [:div.open-bounty-item
-     [:div.open-bounty-item-content
-      [:div.header issue-link]
-      [:div.bounty-item-row
-       [:div.time (human-time updated)]
-       [:span.bounty-repo-label repo-link]]
+        full-repo                   (str owner "/" repo-name)
+        repo-url                    (str "https://github.com/" full-repo)
+        repo-link                   [:a {:href repo-url} full-repo]
+        issue-link                  [:a
+                                     {:href (issue-url owner repo-name issue-number)}
+                                     issue-title]
+        open-claims-click           (fn []
```

Comment:
`issue_id` should be used instead of `issue_number`. Former is unique across GitHub, latter will repeat across repos.

Reply:
@siphiuel nice find, thanks for spotting this. I believe what happened here is that I introduced `open-claims-click`, `close-claims-click`, and `matches-current-issue?` and then vertically aligned the entire let binding (sorry, force of habit). So I actually didn't introduce the issue-link here even though it showed up in my diff for formatting reasons. 

I am, of course, happy to make the refactor as you suggest. But I believe in this instance `issue-number` is working as expected since the `issue-url` function will scope its result by the `repo-name`: ensuring a unique link. 

Let me know if I'm missing something and you'd like to see the refactor. Thanks again for the review!

Reply:
Thank you for response! And apologies @rcullito - I've pointed to a wrong code line :(. I meant that using `issue_number` in body of `open-claims-click` as a parameter to `:open-bounty-claim` event is not good as it will not be unique across repos. Please let me know if this is clearer now, or if I failed again with my English:(

Reply:
We end up storing issue numbers in `(:open-bounty-claims db)`, while we should put issue ids there.

Reply:
Oh, ok. Yes that makes total sense! That's a really good spot. Thank you for clarifying @siphiuel ! I see what you mean 👍 I can definitely make that change.